Libraries/Text/TextInput/Multiline/RCTMultilineTextInputViewManager.m

```diff
@@ -21,5 +21,6 @@ - (RCTUIView *)view // TODO(macOS ISS#3536887)
 
 RCT_REMAP_NOT_OSX_VIEW_PROPERTY(dataDetectorTypes, backedTextInputView.dataDetectorTypes, UIDataDetectorTypes) // TODO(macOS GH#774)
 RCT_REMAP_OSX_VIEW_PROPERTY(dataDetectorTypes, backedTextInputView.enabledTextCheckingTypes, NSTextCheckingTypes) // TODO(macOS GH#774)
+RCT_EXPORT_VIEW_PROPERTY(scrollEnabled, BOOL)
```

Comment:
I guess this is a macOS specific file is why this macro wasn't already here?

Reply:
I think is a shared file by looking at macros like `RCT_REMAP_NOT_OSX_VIEW_PROPERTY` and `RCT_REMAP_OSX_VIEW_PROPERTY `